pandaharvester/harvesterbody/submitter.py

```diff
@@ -300,6 +302,13 @@ def run(self):
                                                     tmpStr = \
                                                         'failed to register a worker for PandaID={0} with batchID={1}'
                                                     tmpLog.error(tmpStr.format(jobSpec.PandaID, workSpec.batchID))
+                                    # enqueue to monitor fifo
+                                    if harvester_config.monitor.fifoEnable \
```

Comment:
Would be better to have

if hasattr(harvester_config.monitor, 'fifoEnable') and harvester_config.monitor.fifoEnable

so that other people can upgrade the code w/o changes to harvester_config.cfg.